### PR TITLE
Fix merge conflicts in prepare_sector_network

### PR DESCRIPTION
**Pull request by tgilon**
## Changes proposed in this Pull Request

This PR fix merge conflicts in `prepare_sector_network`

## Checklist

- [x] I tested my contribution locally and it works as intended.
- [ ] Code and workflow changes are sufficiently documented.
- [ ] Changed dependencies are added to `envs/environment.yaml`.
- [ ] Changes in configuration options are added in `config/config.default.yaml`.
- [ ] Changes in configuration options are documented in `doc/configtables/*.csv`.
- [ ] Sources of newly added data are documented in `doc/data_sources.rst`.
- [ ] A release note `doc/release_notes.rst` is added.

### CODE REVIEW

**Comment by tgilon** [comment]
@daniel-rdt This will be needed in [Form Energy Storage](https://github.com/open-energy-transition/form-energy-storage)

File: scripts/prepare_sector_network.py [hunk 1]
```diff
@@ -1111,13 +1111,8 @@ def add_generation(n, costs, existing_capacities=0, existing_efficiencies=None):
 
     nodes = pop_layout.index
 
-<<<<<<< HEAD
-    fallback = {"OCGT": "gas"}
-    conventionals = options.get("conventional_generation", fallback)
-=======
     conventionals = options["conventional_generation"]
```
Comment (martacki) [inline scripts/prepare_sector_network.py:1114]: Looks to me like HEAD here is better than upstream

```suggestion
        fallback = {"OCGT": "gas"}
        conventionals = options.get("conventional_generation", fallback)
```

Comment (tgilon) [reply scripts/prepare_sector_network.py:1114]: Do we have specific code on this ? Upstream code : https://github.com/PyPSA/pypsa-eur/blob/master/scripts/prepare_sector_network.py#L1114

Comment (martacki) [reply scripts/prepare_sector_network.py:1114]: I guess we previously improved it on our fork. I'd like to keep our changes and then contribute them to upstream but I can't see all the changes we have here now due to the diffusion...

**Pull request merged**